conans/client/conf/config_installer.py

```diff
@@ -151,8 +156,8 @@ def configuration_install(path_or_url, cache, output, verify_ssl, requester,
             _process_zip_file(path_or_url, cache, output, tmp_folder)
         elif config_type == "url":
             args = None
-            _process_download(path_or_url, cache, output, tmp_folder, verify_ssl,
-                              requester=requester)
+            _process_download(
+                path_or_url, cache, output, tmp_folder, verify_ssl, requester=requester)
```

Comment:
Same here, we break arguments when max length is reached, as it was before.

Reply:
I agree with you. I use [yapf](https://github.com/google/yapf) to format my code. Instead of autopep8, yapf tries to keep the code more readable for humans instead of just enforcing the pep8 rule. I gonna create a format config file and add for https://github.com/conan-io/conan/issues/3558